RecoTracker/TkHitPairs/src/HitPairGeneratorFromLayerPair.cc

```diff
@@ -86,26 +86,18 @@ void HitPairGeneratorFromLayerPair::hitPairs(
   }
 }
 
-
-HitDoublets HitPairGeneratorFromLayerPair::doublets( const TrackingRegion& region,
-						    const edm::Event & iEvent, const edm::EventSetup& iSetup, Layers layers) {
-
-  typedef OrderedHitPair::InnerRecHit InnerHit;
-  typedef OrderedHitPair::OuterRecHit OuterHit;
+void HitPairGeneratorFromLayerPair::doublets2(const TrackingRegion& region,
```

Comment:
OK, indeed, the doublets implementation changed. No need for changes related to this comment then.